versions.gradle

```diff
@@ -7,7 +7,7 @@
 
 ext {
     versionCode = 55
-    versionName = '3.1.1'
+    versionName = '3.2.0'
```

Comment:
This should be part of release, not feature, I guess

Reply:
It was a part of your PR that was approved 😉